addons/website_event_track_quiz/controllers/event_track_quiz.py

```diff
@@ -32,6 +33,8 @@ def event_track_quiz_submit(self, event_id, track_id, answer_ids):
         result = {
             'answers': {
                 answer.question_id.id: {
+                    'awarded_points': answer.awarded_points,
+                    'correct_answer': answer.question_id.correct_answer_id.text_value,
```

Comment:
A bit strange to return the correct answer... you receive valid values, so I suppose unlimited tries is actually 2 tries, one to have all right answers, then one to choose the right one ? A bit strange ... Why would we give the right value ?

Reply:
By experimenting a few things, I noticed that this template is not used at all: https://github.com/odoo/odoo/blob/9b01803649d277649afba67ee42d15dd8d6fff00/addons/website_event_track_quiz/static/src/xml/quiz_templates.xml#L3~L42 It might be great to delete it to avoid sending it to the client.